src/codegen/enums.rs

```diff
@@ -0,0 +1,154 @@
+use analysis::namespaces;
+use case::CaseExt;
+use codegen::general::{self, version_condition, version_condition_string};
+use env::Env;
+use file_saver;
+use library::*;
+use std::collections::HashSet;
+use std::io::prelude::*;
+use std::io::Result;
+use std::path::Path;
+use traits::*;
+use version::Version;
+
+pub fn generate(env: &Env, root_path: &Path, mod_rs: &mut Vec<String>) {
+    let path = root_path.join("enums.rs");
+    mod_rs.push("\nmod enums;".into());
+    file_saver::save_to_file(path, env.config.make_backup, |w| {
+        try!(general::start_comments(w, &env.config));
+        try!(writeln!(w, "{}", r#"
```

Comment:
This really needed be raw string? For enums.rs changing all raw to normal seems has no effect on result file.
Also mixing multiline strings starting from beginning of line and not slightly confusing.